*: fix typos reported by Hunspell (#5848)
tldr-pages/tldr: pages/common/sonar-scanner.md

# sonar-scanner

> > SonarScanner is a generic scanner for SonarQube for projects do not use any specific build tool like maven, gradle, etc.

Reviewer: ```suggestion
> SonarScanner is a generic scanner for SonarQube for projects do not use any specific build tool like Maven, Gradle, etc.
```

Reviewer: Is it me... or this sentence does not make any sense? :innocent:

Reviewer 2: ```suggestion
> SonarScanner is a generic scanner for SonarQube for projects, which do not use any specific build tool like maven, gradle, etc.
```
Still not happy with the `for SonarQube for projects`, but at least it's a bit better now

Author: You guys are both totally right — I didn't even pay a speck of attension to the rest of the sentence until _just_ now 😄

Author: This is much better, right?
```suggestion
> SonarScanner is a generic scanner for SonarQube projects that do not use build tools such as Maven, Gradle, or Ant.
```

Reviewer 3: Hrm, this is an awkward one - I completely agree on the double `for` there. What about this?
```suggestion
> SonarScanner is a generic SonarQube scanner for projects that do not use any specific build tool such as Maven or gradle.
```

Ref https://docs.sonarqube.org/latest/analysis/scan/sonarscanner/ for additional docs on the thing I think

Reviewer: `gradle`-> `Gradle` 😇
